indexing-service/src/main/java/org/apache/druid/indexing/common/task/AbstractBatchIndexTask.java

```diff
@@ -935,9 +935,34 @@ protected TaskReport.ReportMap buildLiveIngestionStatsReport(
     );
   }
 
+  protected TaskReport.ReportMap buildLiveIngestionStatsReportWithoutContext(
```

Comment:
Does the live report really need the context?
For a running task, we can always check the task payload to get the context?

If we do need the report, rather than calling the new method `xyzWithoutContext`, we should call the older method `buildLiveIngestionStatsAndContextReport()`.

Reply:
We probably don't need it, to be honest. I think we could just drop it for now.

Reply:
Done